### PR TITLE
Split deployer remove command in two and output receipt.json to config/

### LINKED ISSUE #855: Improve Docker image interfaces
Documentation for https://github.com/api3dao/airnode/issues/1189

### PR DESCRIPTION
**Pull request by dcroote**
Documentation for api3dao/airnode#1322.

Rather than one clunky `remove` command, there are now two remove commands.  `remove-with-receipt` (preferred) uses `receipt.json` while `remove-with-deployment-details` expects the various deployment details instead.

The `receipt.json` default output folder is now `config/` rather than `output/` and therefore all references to the `output/` configuration directory have been removed or replaced, including in the `deploy` command. 

Closes #855

### CODE REVIEW

**Comment by amarthadan** [comment]
Please, link the https://github.com/api3dao/api3-docs/issues/855 so it can be closed by merging this PR.

File: docs/airnode/v0.8/grp-providers/docker/deployer-image.md [hunk 1]
```diff
@@ -100,7 +94,6 @@ docker run -it --rm \
   --env-file aws.env \
   -e USER_ID=$(id -u) -e GROUP_ID=$(id -g) \
   -v "$(pwd)/config:/app/config" \
-  -v "$(pwd)/output:/app/output" \
   api3/airnode-deployer:0.7.2 deploy
```
Comment (amarthadan) [inline docs/airnode/v0.8/grp-providers/docker/deployer-image.md]: This is the documentation for v0.8, right? Shouldn't there be a `0.8.0` version (even though not released yet)?

File: docs/airnode/v0.8/reference/packages/deployer.md [hunk 2]
```diff
@@ -108,26 +108,9 @@ cp config/secrets.env.example config/secrets.env
...
-
-1. Make sure you have `config.json` and `secrets.env` ready. Then, use the
-   `deploy` command to trigger your first deployment.
-2. In order to update the Airnode configuration:
-   - Update the `config.json` and `secrets.env` files as needed.
-   - Run the `deploy` command again.
-3. Use the `remove` command to remove the Airnode deployment. Use the `-r`
-   option to provide the receipt file from the latest deployment or manually add
-   the required arguments.
-
```
Comment (amarthadan) [inline docs/airnode/v0.8/reference/packages/deployer.md:129]: Is this no longer needed?

**Review by amarthadan — CHANGES_REQUESTED** [review]

**Comment by wkande** [comment]
I changed the TOC level from [2,3] to [2,4] for `reference > packages > deployer.md`. This way the terms `deploy` and `remove-<x>` will be easier to find in a search.

File: docs/airnode/v0.8/reference/packages/deployer.md (hunk 2 above)
Comment (wkande) [reply docs/airnode/v0.8/reference/packages/deployer.md:129]: So is this resolved? @dcroote @amarthadan

Comment (amarthadan) [reply docs/airnode/v0.8/reference/packages/deployer.md:129]: Not yet, I've asked just this morning...my morning :smile: So, @dcroote is probably asleep and will respond in the morning...his morning :smile:

File: docs/airnode/v0.8/grp-providers/docker/deployer-image.md (hunk 1 above)
Comment (dcroote) [reply docs/airnode/v0.8/grp-providers/docker/deployer-image.md]: Correct, but @wkande updates all of the version instances when releasing a new version so I didn't update these. @wkande if you'd prefer me update these let me know.

File: docs/airnode/v0.8/reference/packages/deployer.md (hunk 2 above)
Comment (dcroote) [reply docs/airnode/v0.8/reference/packages/deployer.md:129]: I augmented the subsequent individual sections on `Airnode Deployment` and `Airnode Removal`, making this redundant.

File: docs/airnode/v0.8/grp-providers/docker/deployer-image.md (hunk 1 above)
Comment (wkande) [reply docs/airnode/v0.8/grp-providers/docker/deployer-image.md]: @dcroote If you want to change them that is OK. I always double check them when I release. Probably a better workflow than the what I have done in the past to be honest. I suppose it helps others to see the change sooner than later.  I'll hold off the merge.

Comment (dcroote) [reply docs/airnode/v0.8/grp-providers/docker/deployer-image.md]: Done! I didn't update links to the v0.8 quick deploy zip files nor to airnode repo files with the v0.8 tag (e.g. (https://github.com/api3dao/airnode/tree/v0.8/packages/...) as neither of these exist yet.

**Comment by dcroote** [comment]
I updated the version in the deployer commands to `0.8.0` and also linked this PR to the appropriate issue so I think this should now be good to go.

**Review by amarthadan — APPROVED** [review]

**Comment by martinkolenic** [comment]
Hi @dcroote, I've gone through the package description in the docs and that seems to be okay; however, you might want to link to it from the [Airnode Deployer Image](https://api3-docs--improve-deployer-ux-0soa1cyr.web.app/airnode/v0.8/grp-providers/docker/deployer-image.html) page so that the user can quickly reference it in case they accidentally delete the receipt or something else goes wrong with it. The deployer image page does not mention anything other than this or manual removal, so it might be good to mention that as well.

**Pull request merged**